### PR TITLE
Update v.json for VvPhpDollar plugin

### PR DESCRIPTION
**Pull request by ZhaonanLi**
Submit VvPhpDollar plugin from Zhaonan Li.

### CODE REVIEW

File: repository/v.json [hunk 1]
```diff
@@ -483,6 +483,17 @@
 					"branch": "master"
 				}
 			]
+		},
+		{
+			"name": "VvPHPDollar",
+			"details": "https://github.com/ZhaonanLi/VvPhpDollar/releases/tag/1.0.0",
```
Comment (FichteFoll) [inline repository/v.json]: This should be `"https://github.com/ZhaonanLi/VvPhpDollar"`.

Comment (ZhaonanLi) [inline repository/v.json]: Hi FichteFoll,

I have modified the code and the v.json file.
Thanks a lot!

Sincerely,
Zhaonan Li.

**Comment by ZhaonanLi** [comment]
Hi FichteFoll,

I have modified the code and the v.json file.
Thanks a lot!

Sincerely,
Zhaonan Li.

**Comment by FichteFoll** [comment]
Alright.

You should remove `Default.sublime-keymap` and `Default.sublime-commands` though, they will fail anyway because your command expects two paramters.

**Comment by ZhaonanLi** [comment]
Hi FichteFoll,

Thank you so much!
I have **removed** `Main.sublime-menu`, `Default.sublime-commands`, `Default.sublime-keymap`, and **updated** the `README.md`, then **created** a new `release`(`1.0.1`) .

**But** it seems that the version of plugin is still `1.0.0` on [Package Control][pk-ctrl] , what else should I do to update the plugin?

Thanks again!

Sincerely,
Zhaonan Li.

[pk-ctrl]: https://packagecontrol.io/packages/VvPHPDollar "Package Control"

**Comment by ZhaonanLi** [comment]
Hi FichteFoll,

The version of plugin has been updated to `1.0.1` on [Package Control][pck-ctrl].

**But** I  notice that after installing the `VvPhpDollar`, via [Package Control][pck-ctrl], the plugin will not work, if user doesn't **restart** the `sublime text`. 

Is there some method to let `sublime text` load the new-installed plugin **automatically** (without **restart**)?

Sincerely,
Zhaonan Li.

[pck-ctrl]: https://packagecontrol.io/packages/VvPHPDollar "Package Control"

**Comment by ZhaonanLi** [comment]
Hi FichteFoll,

I tried several times (`remove it` + `install it`), now it seems that the `sublime text 2 and  3` can load it **automatically**(without **restart**).

I didn't figure out the **reason**.

Sincerely,
Zhaonan Li.

**Comment by FichteFoll** [comment]
It could have been a bug with Package Control. Which version of it do you have installed? (*Package Control: List Packages* command)

**Comment by ZhaonanLi** [comment]
Hi FichteFoll,

The current version I installed is 1.0.1, and it seems sublime text can load it **automatically** without **restart**.

Sincerely,
Zhaonan Li.

**Comment by FichteFoll** [comment]
I was talking about Package Control's version, but it doesn't really matter anyway.

**Comment by ZhaonanLi** [comment]
Okay! Thanks FichteFoll.

**Pull request merged**